mobile/examples/phi-3-vision/android/android/app/src/main/java/ai/onnxruntime/genai/vision/demo/GenAIImage.java

```diff
@@ -0,0 +1,60 @@
+package ai.onnxruntime.genai.vision.demo;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import ai.onnxruntime.genai.GenAIException;
+import ai.onnxruntime.genai.Images;
+
+public class GenAIImage {
+    Images images = null;
+    Bitmap bitmap = null;
+
+    GenAIImage(Context context, Uri uri, final int maxWidth, final int maxHeight) throws IOException, GenAIException {
+        Bitmap bmp = decodeUri(context, uri, maxWidth, maxHeight);
+        String filename = context.getFilesDir() + "/multimodalinput.png";
+        FileOutputStream out = new FileOutputStream(filename);
+        bmp.compress(Bitmap.CompressFormat.PNG, 100, out); // bmp is your Bitmap instance
+        // PNG is a lossless format, the compression factor (100) is ignored
+        images = new Images(filename);
```

Comment:
FYI, was running into an error here:
`java.lang.UnsatisfiedLinkError: No implementation found for long ai.onnxruntime.genai.Images.loadImages(java.lang.String) (tried Java_ai_onnxruntime_genai_Images_loadImages and Java_ai_onnxruntime_genai_Images_loadImages__Ljava_lang_String_2) - is the library loaded, e.g. System.loadLibrary?`

this ort-genai PR should fix it:
https://github.com/microsoft/onnxruntime-genai/pull/797